libraries/messages/src/sensor.rs

```diff
@@ -18,6 +18,34 @@ pub struct Sensor {
 #[cfg_attr(feature = "ts", ts(export))]
 pub enum SensorData {
     Sbg(Sbg),
+    SbgShort(SbgShort),
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, Format)]
+#[cfg_attr(feature = "ts", derive(TS))]
+#[cfg_attr(feature = "ts", ts(export))]
+pub struct SbgShort {
```

Comment:
Leave it like this for now, but since we'll have a communication board, all of our messages will need to be lesser than 64 bytes.

Reply:
Agreed. It could change to SBGNav, SBGAir, etc to send different data in smaller payloads.